ui/src/app/page.tsx

```diff
@@ -71,7 +71,7 @@ async function CrudShowcase() {
   return (
     <div className="w-full max-w-xs">
       {latestPost ? (
-        <p className="truncate">Your most recent post: {latestPost.name}</p>
+        <p className="truncate">Your most recent post: {"aaaa"}</p>
```

Comment:
Intentional?

Reply:
You wrote that when changing the db adapter :)